CHANGELOG.md

```diff
@@ -30,6 +30,12 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
+## 2.1.0a1
+
+**Aug 26, 2024**
+- `Order.get_assets` now allows to get assets from orders in `BEING_FULFILLED` state.
+- `Order.get_assets` now returning an iterator instear of list.
```

Comment:
Can be this considered a breaking change, I think so. Should we bump to major version bump or force the output to list instead of Iterator.

Reply:
We should not make breaking change at this point, let's keep the signature